docs/DeveloperGuide.md

```diff
@@ -394,30 +394,26 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | new user                                    | see usage instructions                                       | refer to instructions when I forget how to use the App                             |
+| `* * *`  | new user                                    | purge all existing data                                      | get rid of the experimental data when exploring the app                            |
+| `* * *`  | lazy tutor                                  | add students by name                                         | include new students into our app                                                  |
+| `* * *`  | user                                        | delete a person                                              | remove entries that I no longer need                                               |
+| `* * *`  | user                                        | find a student by name or subject                            | locate details of persons without having to go through the entire list             |
+| `* * *`  | tutor                                       | update my students details as they progress through learning | keep track of their information e.g. test scores                                   |
+| `* * *`  | tutor                                       | quickly edit student details                                 | save time re-adding students if I have added the wrong details by accident         |
+| `* * *`  | flexible tutor                              | edit my student’s lesson timings                             | accommodate any changes in timing requested by them                                |
+| `* * *`  | private tutor                               | install the app on my device with one click                  | use the app with ease without much trouble building the environment                |
+| `* * *`  | private tutor                               | see a specific student’s data from the contact list          | get a more concise and detailed view of the student                                |
+| `* *`    | user                                        | hide private contact details                                 | minimize chance of someone else seeing them by accident                            |
+| `*`      | tutor                                       | see my teaching schedule                                     | complete lesson preparation that is catered to the student before tuition sessions |         |                                             |                                                              |                                                                                    |
+| `*`      | user with many students in the student list | sort persons by name                                         | locate a person easily                                                             |
```

Comment:
Should we include functions that we don't currently have in the DG user stories? I don't think we have the sorting functionality for names.

Reply:
Okies, I have made the necessary changes in the recent commit, thank youu